Classes/Controller/FrontEndEditorController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Tea\Controller;
+
+use Psr\Http\Message\ResponseInterface;
+use TTN\Tea\Domain\Repository\Product\TeaRepository;
+use TYPO3\CMS\Core\Context\Context;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Mvc\Controller\ActionController;
+
+/**
+ * Controller for a CRUD FE editor for teas.
+ */
+class FrontEndEditorController extends ActionController
+{
+    private TeaRepository $teaRepository;
+
+    public function injectTeaRepository(TeaRepository $teaRepository): void
+    {
+        $this->teaRepository = $teaRepository;
+    }
```

Comment:
Is there a reason not use constructor injection?
The `ActionController` has moved everything to method injection in order to free constructor injection for final controller.

Reply:
Thanks, good point! Done.